modules/tensor_mechanics/src/materials/ComputeStrainBase.C

```diff
@@ -55,6 +55,16 @@ ComputeStrainBase::ComputeStrainBase(const InputParameters & parameters)
     mooseError(
         "The number of variables supplied in 'displacements' must match the mesh dimension.");
 
+  if (_ndisp == 1 && _volumetric_locking_correction)
+    mooseError("Volumetric locking correction have to be set to false for 1-D problems.");
+
+  if (getParam<bool>("use_displaced_mesh"))
```

Comment:
I recall that there was some good reason we were doing this in `initialSetup()` rather than the constructor. Did you need to move them?

Reply:
Never mind.